src/swsssdk/dbconnector.py

```diff
@@ -67,6 +67,12 @@ def get_instance(db_name):
         SonicDBConfig.inst_name_validation(inst_name)
         return SonicDBConfig._sonic_db_config["INSTANCES"][inst_name]
 
+    @staticmethod
+    def get_instance_list():
```

Comment:
>get_instance_list [](start = 8, length = 17)

rename get_instancelist so it follow get_dblist

Reply:
DONE